<launch>

    <arg name="twist_topic"         default="twist_controller/command"/>
    <arg name="pose_topic"          default="target_frame"/>
    <arg name="wrench_topic"        default="target_wrench"/>
    <arg name="frame_id"            default="vx300s/base_link"/>
    <arg name="end_effector"        default="vx300s/ee_gripper_link"/>
    <arg name="publishing_rate"     default="50"/>

    <!-- Numerically integrate twists into a pose -->
    <node name="twist_converter" pkg="morpheus_teleop" type="twist_converter.py" output="screen">
        <param name="twist_topic"       value="$(arg twist_topic)"/>
        <param name="pose_topic"        value="$(arg pose_topic)"/>
        <param name="wrench_topic"      value="$(arg wrench_topic)"/>
        <param name="frame_id"          value="$(arg frame_id)"/>
        <param name="end_effector"      value="$(arg end_effector)"/>
        <param name="publishing_rate"   value="$(arg publishing_rate)"/>
    </node>

</launch>